src/commander/log.rs

```diff
@@ -212,7 +212,7 @@ impl Commander {
                         "obslog",
                         "--no-graph",
                         "--template",
-                        r#"change_id ++ "\n""#,
+                        r#"commit.change_id() ++ "\n""#,
```

Comment:
I guess you also need to update minimum jj version (it's a constant defined somewhere, don't remember where exactly)

Reply:
Oh, oops, I only thought about the `--config-toml` change, but the template language change actually needs 0.33, I'll update that.

Reply:
Probably then do not restrict that much? 0.33 was released a week ago